joint_state_publisher_gui/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(joint_state_publisher_gui)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+install(PROGRAMS joint_state_publisher_gui/joint_state_publisher_gui DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
Shouldn't the script be placed in `<package_name>/scripts/<script_name>`, not `<package_name>/<package_name>/<script_name>`?

Reply:
Sure.  Fixed in 1db983d.